### PR TITLE
Setup prettier

### PR DESCRIPTION
**Pull request by Rachit1313**
Fixes a part of #4

### CODE REVIEW

File: yarn.lock
Comment (corners2wall) [inline yarn.lock:1]: No need to delete this file. 
This project uses yarn as package manager

File: package-lock.json
Comment (corners2wall) [inline package-lock.json:1]: Please delete this file.

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules/
+package.json
+package-lock.json
```
Comment (corners2wall) [inline .prettierignore]: @Rachit1313 please add EOF and bellow 👇

**Comment by Rachit1313** [comment]
Is this what you mean, not sure, Sorry @corners2wall

**Comment by corners2wall** [comment]
> Is this what you mean, not sure, Sorry @corners2wall

:rofl:  no, bro. EOF - end of file just empty line in your file for not see this: 
Screenshot from 2023-10-29 17-55-24
You can read: [this](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)

**Comment by Rachit1313** [comment]
done, sorry

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "endOfLine": "lf",
+  "insertPragma": false,
+  "proseWrap": "preserve",
+  "requirePragma": false,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "trailingComma": "es5",
+  "useTabs": false,
+  "printWidth": 100
+}
```
Comment (corners2wall) [inline .prettierrc]:
```suggestion
}

```

**Comment by corners2wall** [comment]
@Rachit1313  Seems good

**Comment by Rachit1313** [comment]
> @Rachit1313 Seems good

Are you planning to merge it?

**Comment by corners2wall** [comment]
@Rachit1313 Yes

**Comment by corners2wall** [comment]
Good job, keep going growing  📈

**Pull request merged**